.drone.star

```diff
@@ -26,192 +22,161 @@ def testing(ctx):
 
     repo_slug = ctx.build.source_repo if ctx.build.source_repo else ctx.repo.slug
 
-    return {
-        'kind': 'pipeline',
-        'type': 'docker',
-        'name': 'testing',
-        'platform': {
-            'os': 'linux',
-            'arch': 'amd64',
+    lint_test_build = [
+        {
+            'name': 'clone',
+            'image': 'owncloudci/alpine:latest',
+            'commands': [
+                'git clone https://github.com/%s.git .' % (repo_slug),
+                'git checkout $DRONE_COMMIT',
+            ]
         },
-        "clone": {
-            # The default drone clone step "internally auto-merges" the PR branch to master.
-            # That confuses SonarCloud. The analysis report does not come from the actual
-            # real commit at the tip of the branch.
-            "disable": True,
```

Comment:
This needs to be kept. The reason behind it is sonarcloud. Normally, drone tries so "pre-merge" the pr branch into master. For the unit tests and the upload of the coverage, we need to disable the default drone "clone and merge" routine and do the clone manually.

Reply:
Should just have "changed line numbers" and can be found lower in the file :)